### ISSUE TITLE
Installation?

### ISSUE THREAD

**Issue by FluxAugur** [bojjenclon/FoundryVTT-Ironsworn#1]
How do you get this to install?

**Comment by bojjenclon** [comment]
Sorry about the lack of instructions, I never really expected anyone else to use this so I never bothered putting together a README.

The basic setup goes like this:

1. Clone the repo into a folder called "ironsworn" (e.g. `git clone https://github.com/bojjenclon/FoundryVTT-Ironsworn ironsworn`)
2. npm install
3. npm run update
4. Create a file called foundryconfig.json in the root of this project (file example below)
5. npm run build

`npm run build` will create a symbolic link from the build directory of the project to your Foundry systems directory, so you'll likely need to run this with root/admin access.

---

**foundryconfig.json**
```json
{
  "dataPath": "",
  "repository": "",
  "rawURL": ""
}
```
The only property here that needs to be filled in is "dataPath" - it needs to point to the root of your Foundry folder. In other words, the folder which contains the folders:
* Config
* Data
* Logs

---

I haven't gone through a blank slate setup in a bit, so feel free to reply if something goes wrong as I may have missed a step. I'll try to help out where I can. :)

**Comment by FluxAugur** [comment]
Yup. I got all that from the code. My set up is exactly that, but Foundry doesn’t show the system. I’m having the same problem with a custom module for sharing data between worlds. 🤷🏻‍♂️

**Comment by bojjenclon** [comment]
You've unfortunately caught me at a rather bad time as I've been extremely busy of late, but I'll try to futz around with it when I get a chance and see if I can help at all.

The first thing I'd def check - and I know it sounds simple but foundry can be particular about this so I figured I'd make sure - is that you renamed the cloned folder to just "ironsworn." If its FoundryVTT-Ironsworn it won't work, as all the paths in the system are setup as "ironsworn" and Foundry needs to see that path specifically.
